rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -1,87 +1,41 @@
 [id="rosa-release-notes"]
-= {product-title} 4.10 release notes
+= What's new with {product-title}
 include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: rosa-release-notes
 
 toc::[]
 
 {product-title} (ROSA) is a fully-managed, turnkey application platform that allows you to focus on delivering value to your customers by building and deploying applications. Red Hat and AWS site reliability engineering (SRE) experts manage the underlying platform so you do not have to worry about the complexity of infrastructure management. ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to further accelerate the building and delivering of differentiating experiences to your customers.
 
-[id="about-this-release-section_{context}"]
-== About this release
+{product-title} clusters are available on the link:https://console.redhat.com/openshift[Hybrid Cloud Console]. With the Red Hat OpenShift Cluster Manager application for ROSA, you can deploy {product-title} clusters to either on-premises or cloud environments.
 
-{product-title} https://access.redhat.com/errata/RHEA-2022:2252[RHEA-2022:2252] is now available. This release uses Kubernetes 1.24 with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} 4.10 are included in this topic.
+[id="updating_{context}"]
+== Updating the ROSA CLI tool
 
-{product-title} clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for ROSA, you can deploy {product-title} clusters to either on-premise or cloud environments.
+You can download the latest version of the ROSA CLI from the Hybrid Cloud Console.
 
-[id="about-this-release_{context}"]
-== ROSA layered and dependent component support and compatibility
-The scope of support for layered and dependent components of ROSA changes independently of the ROSA version. To determine the current support status and compatibility for an add-on, see the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle].
+To update to the latest version of the ROSA CLI:
 
-[id="rosa-new-features_{context}"]
-== New features and enhancements
+. Download the file from the link:https://console.redhat.com/openshift/downloads[Hybrid Cloud Console].
+. Unzip the downloaded file.
+. In the terminal, type `sudo mv rosa /usr/bin/rosa` then press **Enter**.
+//. In the terminal, type `chmod a+x /usr/bin/rosa` to make the ROSA binary you downloaded executable.
+. Type `rosa version` in the terminal to confirm your version. "Your ROSA CLI is up to date." is displayed when you have updated.
 
-[id="rosa-product-enhancements_{context}"]
-=== Product enhancements
+[id="rosa-new-changes-and-updates_{context}"]
+== New changes and updates
 
-The following list describes the key features of ROSA:
+[id="rosa-q1-2023_{context}"]
+=== Q1 2023
```

Comment:
Consider reorganizing the headers under this Q1 2023 header, which should be H2. Right now, the new changes are H2 and above this Q1 2023 header, which is confusing and doesn't read very well.  Additionally, the new features and enhancements, known issues, and any bug fixes should be under the Q1 2023 header for ease of reading. I suggest using the same format and wording as we use in our [OCP release notes](https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html) or [ROSA serverless release notes](https://docs.openshift.com/rosa/serverless/serverless-release-notes.html). 

The TOC should look something like this:

```
Q1 2023
  New features and enhancements
  Deprecated and removed features
  Known Issues
```
	
I know your'e trying to do something a little different than release notes, but I'm assuming this page will updated quarterly at the very least, so it's important to get it organized now in a way that's easy to understand as well as modify later.

Reply:
Left the structure as is, as it lends itself better to the continuous delivery release cadence that ROSA has versus OCP and the versioned releases.